ui.frontend/src/main/webpack/site/styles/wkndsample/README.md

```diff
@@ -0,0 +1,110 @@
+# WKND sample styleguide
+
+The information provided here aims at showing how to structure and implement your styles. However, feel free to organize the way that fits your project.
+
+## Table of Contents
+
+- [Files](#files)
+- [Classes](#classes)
+
+## Files
+
+Use the following files in order to update according to your UI needs. Filenames are self-explanatory.
+
+- \_animation.scss
+- \_breakpoints.scss
+- \_colors.scss
+- \_media_queries.scss
+- \_spacing.scss
+- \_typography.scss
+
+They are all imported in \_wkndsample.scss.
+
+## Classes
+
+Below are all the classes that are used in the SAAS module and can be styled.
+
+### Form and search input
+
+```
+.cmp-saas__form {}
+.cmp-saas__search-input-wrapper {}
+.cmp-saas__search-input {}
+```
+
+### Results container
+
+```
+.cmp-saas__results {}
+.cmp-saas__results--show {}
+.cmp-saas__results--hide {}
+```
+
+### Results items container
+
+```
+.cmp-saas__results-items {}
+```
+
+### Results items
+
+```
+.cmp-saas__results-item {}
+.cmp-saas__results-item--best-best {}
+.cmp-saas__results-item-description {}
+.cmp-saas__results-item-title {}
+.cmp-saas__results-item-url {}
+.cmp-saas__results-item-link-text {}
+```
+
+### Search tab
+
+```
+.cmp-saas__tab {}
+```
+
+### Search and load more buttons
+
+```
+.cmp-saas__button {}
+.cmp-saas__load-more-button {}
+```
+
+### Autosuggest
+
+For when query is mispelled. The application will offer alternatives words.
+
+```
+cmp-saas__autosuggest {}
+```
+
+### Not found result
+
+```
+cmp-saas__not-found {}
+```
+
+### Typeahead suggestion
+
+```
+#suggestions {}
```

Comment:
can we prefix this id as well? would like to avoid clashing IDs

Reply:
Sure, will do.

Reply:
@kevinolivar , sorry just noticed: to be consistent between backend and frontend we should use `autocomplete` here...

Reply:
consistency :D the usual...common terminologies.